punx/github_handler.py

```diff
@@ -81,20 +95,32 @@ def get_GitHub_credentials(creds_file_name=""):
     GitHub credentials. Be sure to make the file readable only by the user and
     not others.
 
-    Returns None if credentials are not found per above search.
+    Returns ``None`` if credentials are not found per above search.
     """
-    for variable in ["GH_TOKEN", "GITHUB_TOKEN"]:
-        if variable in os.environ:
-            return os.environ[variable]
-
-    for name in [
-        creds_file_name,
-        os.path.join(os.path.dirname(__file__), CREDS_FILE_NAME),
-        os.path.join(os.path.expanduser("~"), ".config", CREDS_FILE_NAME),
-        os.path.join(os.path.expanduser("~"), ".config", "punx", CREDS_FILE_NAME),
-    ]:
-        if os.path.exists(name):
-            return open(name, "r").read().strip().split()[-1]
+    home = pathlib.Path.home()
```

Comment:
This line will raise an uncaught error if the home cannot be resolved, but the docs say that this function should return a token from environment variables regardless of whether there is a home directory.

https://docs.python.org/3/library/pathlib.html#pathlib.Path.home